### PR TITLE
Add issue count to docs

### LINKED ISSUE #562: Display number of issues for function or tag
Next to the `edit` button on cfdocs is the `issue` button where you can create a new issue. Currently we have more than one issue page and you have to scroll up and down to find a suitable 'project' to solve or improve.

It would be nice to split the button into something like "[`(5) issues`](https://github.com/foundeo/cfdocs/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20cfloop)" and "[`+ issue`](https://github.com/foundeo/cfdocs/issues/new?title=cfloop)". Is there a way to find out the number of issues concerning a specific function or tag? Than we could link to the filtered issues as I linked them. So you can

* Edit the tag or function
* Nag that there is something wrong/Discuss an improvement proposal
* Get a read where to contribute to improve this tag or function

### PR DESCRIPTION
**Pull request by shaedrich**
Resolve #562

### CODE REVIEW

**Comment by shaedrich** [comment]
As you can see [here](https://api.github.com/search/issues?q=is:open+repo:foundeo/cfdocs+sdfghjrctztg) `total_count` will be `0` when no records were found.

File: views/doc.cfm [hunk 1]
```diff
@@ -96,7 +96,10 @@
 				<li role="separator" class="pull-right divider"></li>
 			</cfif>
 			<li class="pull-right">
-				<a href="https://github.com/foundeo/cfdocs/issues/new?title=#encodeForURL(data.name)#" rel="nofollow" class="label label-warning" title="Report an Issue">Issue</a>
+		    		<span class="label label-warning">
+					<a href="https://github.com/foundeo/cfdocs/issues?q=is:issue%20is:open%20#encodeForURL(data.name)#" class="issuecount badge">4</a>
```
Comment (pfreitag) [inline views/doc.cfm]: Was talking about the `4` you can see here... the issue count will be 4 until the JS executes.

File: assets/script.js [hunk 2]
```diff
@@ -74,3 +74,11 @@ _gaq.push(['_trackPageview']);
...
   var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
 })();
+
+//github api requests
+if(!!$('.issuecount').length) {
+	docname = $('ol.container').data('doc');
+	$.getJSON('https://api.github.com/search/issues?q=is:open+repo:foundeo/cfdocs+' + docname,function(res) {
+		$('.issuecount').html(res.total_count);
+	});
+}
```
Comment (pfreitag) [inline assets/script.js]: The JS should be inside the `$(document).ready( function() {` _put in here_ `}`

File: views/doc.cfm (hunk 1 above)
Comment (shaedrich) [reply views/doc.cfm]: Oh, I totally forgot to delete it. Maybe the badge should be hidden when no issues are found?

File: assets/script.js [hunk 3]
```diff
@@ -74,3 +74,13 @@ _gaq.push(['_trackPageview']);
   ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
   var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
 })();
+
+//github api requests
+$(document).ready( function() {
```
Comment (pfreitag) [inline assets/script.js]: This can go inside the `$(document).ready( function() {` on the top of the file, no need to attach a second event handler.

**Comment by pfreitag** [comment]
Awesome thanks

**Comment by shaedrich** [comment]
It was my pleasure

**Pull request merged**